src/main/java/com/frameworkium/core/common/properties/Property.java

```diff
@@ -39,7 +39,9 @@
     FIREFOX_PROFILE("firefoxProfile"),
     VIDEO_CAPTURE_URL("videoCaptureUrl"),
     CHROME_USER_DATA_DIR("chromeUserDataDir"),
-    CUSTOM_BROWSER_IMPL("customBrowserImpl");
+    CUSTOM_BROWSER_IMPL("customBrowserImpl"),
+    REUSE_BROWSER("reuseBrowser"),
+    THREAD("threads");
```

Comment:
I can't remember if there was a reason why we didn't have this before...
Do tests without `reuseBrowser` still work with multiple threads?
I also think this should probably be called `THREADS` or `THREAD_COUNT` and we can also update `AllureProperties.java` to use it.

Reply:
I'll rename to THREADS. The AllureProperties is probably a different ticket.